samples/xmpsample.cpp

```diff
@@ -183,25 +183,25 @@ try {
 
     // -------------------------------------------------------------------------
     // Output XMP properties
-    for (Exiv2::XmpData::const_iterator md = xmpData.begin(); 
-         md != xmpData.end(); ++md) {
-        std::cout << std::setfill(' ') << std::left
-                  << std::setw(44)
-                  << md->key() << " "
-                  << std::setw(9) << std::setfill(' ') << std::left
-                  << md->typeName() << " "
-                  << std::dec << std::setw(3)
-                  << std::setfill(' ') << std::right
-                  << md->count() << "  "
-                  << std::dec << md->value()
-                  << std::endl;
+    for (Exiv2::XmpData::const_iterator md = xmpData.begin();
+	 md != xmpData.end(); ++md) {
+	std::cout << std::setfill(' ') << std::left
+		  << std::setw(44)
```

Comment:
It seems that your editor is replacing here spaces by tabs, was this intended?

Reply:
Nope, Emacs seems to have a very peculiar interpretation of `(setq-default indent-tabs-mode nil)`. I'll fix that.